discover/portal/articles/210-setting-up/01-system-settings/01-system-settings.markdown

```diff
@@ -0,0 +1,121 @@
+*Configuration*. You'll see the same Enable Comments checkbox, and its default
+is now false (unchecked). Checking the box in the Widget Configuration screen
+breaks its link with the System Settings entry. Changing the System
+Settings configuration does nothing to this Widget Instance anymore.
+
+If you make some regrettable configuration decisions and can't recall exactly
+what you did, start over by clicking the actions button
+(![Actions](../../../images/icon-actions.png)), then clicking *Reset Default
+Values*. 
+
+## Configuration Scope [](id=configuration-scope)
+
+While browsing the categories of System Settings, you'll notice that clicking
+into a topic (for example, Blogs) reveals entries at different scopes.each entry
+has a *Name* and a *Scope*. This scope is different than the overall
+application configuration scope discussed in the introduction to this section.
+This is the System Settings application after all, so all of the configuration
+done here affects the System scope. The Scope field here refers to the scope at 
+which this default, system-wide configuration is overridden. There are four 
+values that you'll see under Scope: 
+
+- *System:* Any configuration made at the system scope here becomes the final
+  value for the application in a system-wide fashion. It affects the whole
+  system and isn't overridden anywhere else. 
+
+    ![Figure 4: Some System Settings entries have a System scope.](../../../images/system-settings-system-scope.png)
+
+- *Virtual Instance Scope:* Making configuration changes at this level affects
+    the entire Virtual Instance. 
+
+    ![Figure 5: Some System Settings entries have a Default Configuration for Application scope.](../../../images/system-settings-application-scope.png)
+
+- *Site Scope* Configuration at this scope is overridden in each site. 
+
+    ![Figure 6: Some System Settings entries have a Default Configuration for All Sites scope.](../../../images/system-settings-site-scope.png)
+
+- *Widget Scope:* Configuration at this scope is overridden in each
+    Widget Instance (like the Blogs example above). 
+
+    ![Figure 7: Some System Settings entries have a Default Configuration for All Instances scope.](../../../images/system-settings-instance-scope.png)
+
+If an application is configurable from Instance Settings and System Settings,
+use System Settings to configure it whenever possible. If you use Instance
+Settings and later want to revert to using the System Settings default
+configurations, look for the *Reset Values* button from the Instance Settings
+entry. 
+
+![Figure 8: Some Instance Settings entries have a *Reset Values* button so you can safely revert your configuration changes, falling back to the System Settings defaults.](../../../images/instance-settings-reset-values.png)
+
+The Reset Values button removes configuration values from the database
+so you can rely on the defaults specified in System Settings. If there is no
```

Comment:
This is incorrect. When resetting the values it goes back to the default configuration shipped with the product.

Reply:
Just to clarify, what happens if you do this?

1. System Settings: make a configuration change to a setting that's Virtual Instance Scoped. 
2. Instance Settings: make a configuration change to the same property in Instance Settings.
3. Instance Settings: hit the Reset Values button.

When you go back to System Settings after this, what do you see? The original OOTB value or the value you first set? I suppose if I was familiar with the appropriate DB tables I'd know the answer to this.